### PR TITLE
Partially format librustc_mir with rustfmt.

### PR DESCRIPTION
**Pull request by Ms2ger**
This commit contains some of the changes proposed by a rustfmt invocation,
chosen based on the fairly non-deterministic metric of how much I liked the
change. I expect we will run rustfmt on this crate again later, probably
accepting more of its changes. For now, this is already an improvement over
the status-quo.

### CODE REVIEW

**Comment by Ms2ger** [comment]
CC @nrc

File: src/librustc_mir/hair.rs [hunk 1]
```diff
@@ -113,45 +113,128 @@ pub struct Expr<'tcx> {
 
 #[derive(Clone, Debug)]
 pub enum ExprKind<'tcx> {
```
Comment (gkoz) [inline src/librustc_mir/hair.rs]: This enum has inflated from 40 lines to 120. Did its author perhaps intend it to fit on a single screen?

Comment (nrc) [inline src/librustc_mir/hair.rs]: https://github.com/nrc/rustfmt/issues/418

**Comment by nrc** [comment]
r+ from me, @nikomatsakis not sure if you want to glance over and see if anything offends you

**Comment by nikomatsakis** [comment]
@bors r+

I don't hate it too much, which is probably the best you can expect. :)

**Pull request merged**